### ISSUE TITLE
Suggestion "Use zipWith" raises Error instead of Warning

### ISSUE THREAD

**Issue by Arminius** [ndmitchell/hlint#28]
Let's take the following example code:

``` haskell
foo :: [[a]] -> [[a]] -> [[a]]
foo list1 list2 = sequence $ zipWith (++) list1 list2
```

Here, HLint spits out the following message:

> sequence.hs:4:19: Error: Use zipWithM
> Found:
>  sequence $ zipWith (++) list1 list2
> Why not:
>  Control.Monad.zipWithM (++) list1 list2
> 
> 1 suggestion

The code compiles just fine, so this should be a Warning, not an Error.
I'm using HLint v1.8.57.
